google/resumable_media/requests/download.py

```diff
@@ -58,12 +62,43 @@ def _write_to_stream(self, response):
 
         Args:
             response (~requests.Response): The HTTP response object.
+
+        Raises:
+            DataCorruption: If the download's checksum doesn't agree with
+                server-computed checksum.
         """
+        md5_hash = hashlib.md5()
+        # TODO: Add support for validating CRC32C for composite objects. Note
```

Comment:
@mfschwartz Please remove the `TODO` from the source and put it in the issue tracker.

Reply:
Done